app/components/works/dates_component.html.erb

```diff
@@ -65,6 +70,10 @@
               id: 'work_created_range_start_day',
               class: "form-control" %>
         </div>
+        <div>
+         <%= check_box_tag 'work[created_range(approx0)]', 1, created_range_start_approximate? %>
```

Comment:
just curious if you tried using a value of `true` instead of `1`, which maybe could then let you do `if uncertain` instead then `if uncertain == '1'` below, which seems a bit clearer

Reply:
Thanks @peetucket, lasted commit now uses `true` instead of 1.